src/App.tsx

```diff
@@ -28,7 +28,7 @@ import { CoreDumpManager } from 'lib/coredump'
 import { UnitsMenu } from 'components/UnitsMenu'
 
 export function App() {
-  useRefreshSettings(paths.FILE + 'SETTINGS')
+  useRefreshSettings(PATHS.FILE + 'SETTINGS')
```

Comment:
Why is SETTINGS a string not a constant?

Reply:
Oh so sorry how did I miss this comment? Yes good idea, I can do that in another tiny PR